resources/views/admin/users/table.blade.php

```diff
@@ -0,0 +1,60 @@
+@if(!$admins->isEmpty())
```

Comment:
Let's call partials with underscore. And more understandable names.
`_users-table.blade.php`